_episodes/04-drawing.md

```diff
@@ -103,8 +99,16 @@ Next, we draw a filled, rectangle on the mask:
 # Draw filled rectangle on the mask image
 rr, cc = skimage.draw.rectangle(start=(357, 44), end=(740, 720))
 mask[rr, cc] = False
+
+# Display mask image
+fig, ax = plt.subplots()
+plt.imshow(mask, cmap='gray')
+plt.show()
 ~~~
-{: .python}
+{: .language-python}
+
+Here is what our constructed mask looks like:
+![Maize image mask](../fig/03-maize-mask.png){: .image-with-shadow}
 
 The parameters of the `rectangle()` function `(357, 44)` and `(740, 720)`, are the coordinates of the 
 upper-left (`start`) and lower-right (`end`) corners of a rectangle in *(y, x)* order. 
```

Comment:
Should we point out the coordinate order more prominently, e.g., in a callout? Perhaps a sentence that explains how _x_ and _y_ correspond to rows and columns could also help learners grasp this concept.

Reply:
This is definitely a source of confusion for those learning about this stuff for the first time, but it is already addressed and emphasised a couple of times in the previous episode ([1](https://datacarpentry.org/image-processing/03-skimage-images/index.html#images-are-represented-as-numpy-arrays) and [2](https://datacarpentry.org/image-processing/03-skimage-images/index.html#access-via-slicing)) so I think we are good here.